### PR TITLE
There may be an extra row in the unordered list

### LINKED ISSUE #94619: There may be an extra row in the unordered list
### Elasticsearch Version

8.8.0

### Installed Plugins

_No response_

### Java Version

java version "17.0.6" 2023-01-17 LTS Java(TM) SE Runtime Environment (build 17.0.6+9-LTS-190) Java HotSpot(TM) 64-Bit Server VM (build 17.0.6+9-LTS-190, mixed mode, sharing)

### OS Version

Darwin MacBook-Pro.local 22.3.0 Darwin Kernel Version 22.3.0: Mon Jan 30 20:38:37 PST 2023; root:xnu-8792.81.3~2/RELEASE_ARM64_T6000 arm64

### Problem Description

This unorder list row statement may need to be revised when I read the CONTRIBUTING doc.
in CONTRIBUTING doc 72-73 rows, like below
* For contributions involving the elasticsearch build you can find (details about the build setup in the
* [BUILDING](BUILDING.md) file

### Steps to Reproduce

It is a document bug.

### Logs (if relevant)

_No response_

### PR DESCRIPTION
**Pull request by jackpan123**
This unorder list row statement may need to be revised when I read the CONTRIBUTING doc. fix #94619

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,8 +69,7 @@ cycle.
 * Lines that are not part of your change should not be edited (e.g. don't format
   unchanged lines, don't reorder existing imports)
 * Add the appropriate [license headers](#license-headers) to any new files
-* For contributions involving the elasticsearch build you can find (details about the build setup in the
-* [BUILDING](BUILDING.md) file
+* For contributions involving the elasticsearch build you can find (details about the build setup in the [BUILDING](BUILDING.md) file)
```
Comment (romseygeek) [inline CONTRIBUTING.md:72]: Can you keep the line break, but just remove the `*` from the beginning of line 73?  We limit line lengths in the code base to make things easier to read.

File: docs/changelog/94620.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pr: 94620
```
Comment (romseygeek) [inline docs/changelog/94620.yaml:1]: I don't think you need a changelog entry for a docs change, so you should be able to delete this file.

**Review by romseygeek — CHANGES_REQUESTED** [review]
Thank you for opening! I left a couple of comments, but we should be good to merge once they are addressed.

**Comment by jackpan123** [comment]
@romseygeek Hi, I added the blank row and deleted  94620.yaml

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -69,8 +69,8 @@ cycle.
...
```
Comment (romseygeek) [inline CONTRIBUTING.md:72]:
```suggestion
* For contributions involving the elasticsearch build you can find (details about the build setup in the [BUILDING]
  (BUILDING.md) file)
```

[the remark this answers is not in the document]
Comment (romseygeek) [reply CONTRIBUTING.md:72]: Sorry, this is what I meant

**Comment by jackpan123** [comment]
@romseygeek I am very sorry for not being able to understand your meaning and causing you trouble.

**Review by romseygeek — APPROVED** [review]
Great, thank you!

**Pull request merged**